airbase/pom.xml

```diff
@@ -452,7 +454,7 @@
                         </systemPropertyVariables>
                         <trimStackTrace>false</trimStackTrace>
                         <runOrder>random</runOrder>
-                        <failIfNoTests>true</failIfNoTests>
+                        <failIfNoTests>${air.test.fail-if-no-tests}</failIfNoTests>
```

Comment:
I am wondering what's the usecase for this would be.

If module has no tests, there is no point in invoking surefire at all.
So I'd want to define `air.skip-tests` or `air.there-are-no-tests` property instead.

If, however, the module has tests, but i want to skip test on per-execution basis, the `-DskipTests` is there.

Thus, we shouldn't add this property as is.

Reply:
Good point, we should just skip the plugin entirely for such modules.